<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>raw图查看</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/03.png</normaloff>:/03.png</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>699</width>
     <height>635</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_chooseImg">
        <property name="text">
         <string>选择图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="ImgLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(234, 255, 231);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_opencv_gray">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>CV_Gray图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_opencv">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>CV_GRBG图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_labelImg_gray16">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>QT灰度16</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_labelImg_gray8">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>QT灰度8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clear">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImgLabel</class>
   <extends>QLabel</extends>
   <header location="global">imglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
